homeassistant/components/isy994/__init__.py

```diff
@@ -220,9 +220,11 @@ async def async_setup_entry(
         numbers = isy_data.variables[Platform.NUMBER]
         for vtype, _, vid in isy.variables.children:
             numbers.append(isy.variables[vtype][vid])
-    if isy.conf[ISY_CONF_NETWORKING]:
+    if (
+        isy.conf[CONFIG_NETWORKING] or isy.conf[CONFIG_PORTAL]
+    ) and isy.networking.nobjs:
```

Comment:
Check for Networking Module or UDI Portal in config, and don't create device if there are no resources to load.